### DISCUSSION TITLE
Won't allow me create shortlink with letters.

### DISCUSSION THREAD

**Discussion by ghost** [YOURLS/YOURLS#3443] [Q&A]
Hello, I've been trying to create shortlinks with letters such as.

Short.link/1500va

But when I save the shortlink it comes out as 'short.link/1500', I've also tried to just use letters but comes up with warning stating that I need to type something. So it only works with numbers and nothing else.

**Comment by dgw** [comment]
Sure sounds like https://github.com/josheby/yourls-additional-charsets in `10` mode (digits only).
